src/main.cpp

```diff
@@ -791,7 +764,7 @@ int main(int argc, char**argv)
   delete cass_comm_monitor; cass_comm_monitor = NULL;
   delete alarm_manager; alarm_manager = NULL;
 
-  SAS::term();
+  delete sas_service; sas_service = NULL;
```

Comment:
Hmm, this delete is probably in the wrong place. We should probably delete the exception handler and health checker after the SAS service. Probably not a big deal if you've checked that memento exits cleanly.

Reply:
Yeah, memento does exit cleanly, so probably going to leave as-is